ArticleTemplates/assets/js/modules/youtube.js

```diff
@@ -79,19 +76,6 @@ function checkForVideos() {
 
```

Comment:
I wonder if this `resize` event listener is required now? Will the `MutationObserver` register a `resize` event? I believe `resize` was meant to pickup a rotation of device